Documentation/contributing/development/contributing_guide.rst

```diff
@@ -95,30 +97,46 @@ requirements have been met:
 
 #. All commits are signed off. See the section :ref:`dev_coo`.
 
-#. Pick the appropriate milestone for which this PR is being targeted to, e.g.
-   ``1.1``, ``1.2``. This is in particular important in the time frame between
-   the feature freeze and final release date.
-
-#. Pick the right release-note label
-
-   +--------------------------+---------------------------------------------------------------------------+
-   | Labels                   | When to set                                                               |
-   +==========================+===========================================================================+
-   | ``release-note/bug``     | This is a non-trivial bugfix                                              |
-   +--------------------------+---------------------------------------------------------------------------+
-   | ``release-note/major``   | This is a major feature addition, e.g. Add MongoDB support                |
-   +--------------------------+---------------------------------------------------------------------------+
-   | ``release-note/minor``   | This is a minor feature addition, e.g. Refactor endpoint package          |
-   +--------------------------+---------------------------------------------------------------------------+
+#. (optional) Pick the appropriate milestone for which this PR is being targeted
+   to, e.g. ``1.1``, ``1.2``. This is in particular important in the time frame
```

Comment:
nit: 

```suggestion
#. (optional) Pick the appropriate milestone for which this PR is being targeted,
   e.g. ``1.1``, ``1.2``. This is in particular important in the time frame
```

I always get mixed up in the "for which ..." style sentence but it's valid. The `to` bit at the end is not necessary though.